python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -696,15 +524,16 @@ def cloneVal(v):
         if isinstance(new_val, Node):
             new_val._setParent(obj)
 
-        obj.setNodeAttribute(_getName(self), new_val)
+        obj.setSchemaAttribute(_getName(self), new_val)
 
         obj._setDirty()
 
     def __delete__(self, obj):
         """
         Delete an attribute from teh Descriptor(?) and Node
         """
-        obj.removeNodeAttribute(_getName(self))
+        raise Exception('Descriptor.__delete__')
+        del obj._data[_getName(self)]
```

Comment:
Won't the `Exception` mean that this will always fail and the `del` isn't called?